lib/index.ts

```diff
@@ -158,15 +153,16 @@ class PWMetrics {
     }
   }
 
-  async launchChrome(): Promise<ChromeLauncher> {
+  async launchChrome(): Promise<LaunchedChrome|Error> {
     try {
-      this.launcher = new ChromeLauncher();
-      await this.launcher.isDebuggerReady();
-      return this.launcher;
-    } catch(e) {
       console.log(messages.getMessage('LAUNCHING_CHROME'));
-      await this.launcher.run();
-      return this.launcher;
+      return this.launcher = await launch({
+        port: this.flags.port,
```

Comment:
+1 faced the same issue. Using some other port - works perfectly.
`pwmetrics http://example.com --port=8080`